src/Mapper/ProgrammesDbToDomain/ProgrammeMapper.php

```diff
@@ -88,7 +87,7 @@ private function getBrandModel(array $dbProgramme): Brand
             $this->getMasterBrandModel($dbProgramme),
             $this->getCategoriesModels('genre', $dbProgramme, 'categories'),
             $this->getCategoriesModels('format', $dbProgramme, 'categories'),
-            ($dbProgramme['firstBroadcastDate'] ? DateTimeImmutable::createFromMutable($dbProgramme['firstBroadcastDate']) : null),
+            $dbProgramme['firstBroadcastDate'],
```

Comment:
Is this correct? I assume if `$dbProgramme['firstBroadcastDate']` isn't truthy, it's null, so no need to specifically set it as null?